src/components/DogList/DogList.css

```diff
@@ -8,7 +8,7 @@
 
 .list-item {
   width: 300px;
-  height: 300px;
+  height: 340px;
```

Comment:
Consider using min-height here to have the card grow with the content or truncate on overflow of the `<p>` if you want the cards to maintain an equal height while handling longer text. 
Example:
`p {white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;}`